soccer/src/soccer/coach/coach_node.cpp

```diff
@@ -0,0 +1,76 @@
+#include "coach_node.hpp"
+
+using std::placeholders::_1;
+
+CoachNode::CoachNode(const rclcpp::NodeOptions& options) : Node("coach_node", options) {
+    coach_pub_ = this->create_publisher<rj_msgs::msg::Coach>("/strategy/coach", 10);
+    playstate_change_timer_ = this->create_wall_timer(
+        100ms, std::bind(&CoachNode::check_for_playstate_change, this));
```

Comment:
this warning seems to want you to use a lambda to call the method you want, like so: https://github.com/RoboJackets/robocup-software/blob/ros2/soccer/src/soccer/radio/radio.cpp#L39

the reason is a little confusing to me, but try this SO post: https://stackoverflow.com/questions/1930903/bind-vs-lambda